tests/Makefile

```diff
@@ -43,3 +43,12 @@ test_homebrew:
 	@mkdir -pv tmp
 	@ruby -n ../bin/gen_tests.rb < homebrew.txt > tmp/homebrew.sh
 	@cd tmp && MSG_PREFIX=":: [homebrew] " sh homebrew.sh
+
+# test_pkgng:
+# test_pkgng: Execute FreeBSD tests within MacOS provided by Github-actions
+.PHONY: test_pkgng
+test_pkgng:
+	@set -u && echo $$CI >/dev/null
+	@mkdir -pv tmp
+	@ruby -n ../bin/gen_tests.rb < pkgng.txt > tmp/pkgng.sh
+	@cd tmp && MSG_PREFIX=":: [pkgng] " sh pkgng.sh
```

Comment:
Before executing the last command (`sh pkng.sh`), can you please try to uninstall `bash`? This is important to make sure our script can run well without `bash`.

This wouldn't be the case for homebrew (https://github.com/icy/pacapt/blob/87a7ee54ae9c46357e204023c5c10bd044078404/tests/Makefile#L41), because there is only `bash3` in MacOS as far as I know. (But I am gonna ensure that.)

Thanks a  lot.

Reply:
Sure

Reply:
See also https://github.com/icy/pacapt/pull/190